pkg/storage/kubernetes/ipam.go

```diff
@@ -557,14 +592,6 @@ func IPManagementKubernetesUpdate(ctx context.Context, mode int, ipam *Kubernete
 			break RETRYLOOP
 		}
 
-		if ipamConf.OverlappingRanges {
```

Comment:
Can you clarify why this change was made - the code is common but was added back tot he Allocate and Deallocate case statements. Now the same code is duplicated in each case. Did this fix some other issue or optimization?